src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -162,7 +162,7 @@ private Slave prepareAgentDocker() throws Exception {
             case NO_INIT:
                 return new DumbSlave("docker",
                             "/home/jenkins/agent",
-                            new SimpleCommandLauncher("docker run -i --rm jenkins/slave:3.29-2 java -jar /usr/share/jenkins/slave.jar"));
+                            new SimpleCommandLauncher("docker run -i --rm jenkins/agent:latest-jdk11 java -jar /usr/share/jenkins/agent.jar"));
```

Comment:
@basil thanks for the PR! Question, is it possible that the jenkins version of the `agent` image could get out of sync with the plugin so that it could not run? Or is that not how it works?

Reply:
Remoting is generally both forward and backward compatible, so this should not be a problem.